language/02-guidelines/01-style/accessibility/index.md

```diff
@@ -32,6 +44,8 @@ When choosing colors and contrast ratios for typography it is important to adher
 
 Colors at a 50 value or lower are not accessible on neutral backgrounds, unless they're displayed at 18 pt or larger. There are no formal color contrast requirements for icons or glyphs that are paired with supporting text. If an icon is used separately from supporting text on a neutral background, use a color value of 50 or higher.
```

Comment:
So the thing is `pt` is an absolute value in CSS that is based on the screen resolution. It only roughly translates to `px`.

http://www.w3.org/Style/Examples/007/units.en.html